themes/bootstrap3/less/vendor/bootstrap/mixins/dropdowns.less

```diff
@@ -0,0 +1,21 @@
+// Dropdowns
+
+// Dropdown link styles mixin, see also vendor/bootstrap/dropdowns.less
+
+.dropdown-menu-ul-links {
```

Comment:
@ckaz, where does this class come from? It seems to be added here but I don't see it anywhere else in the code. Maybe I just don't know enough about how LESS/SCSS works, but just curious... :-)